frontend/console/e2e/cron.spec.ts

```diff
@@ -24,3 +24,46 @@ test('send cron request', async ({ page }) => {
 
   expect(responseJson).toEqual({})
 })
+
+test('submit cron form using ⌘+⏎ shortcut', async ({ page }) => {
+  await navigateToDecl(page, 'cron', 'thirtySeconds')
+
+  await page.locator('input#request-path').focus()
+
+  // The keypress is sometimes flakey in playwright, so try 3 times. Ideally we'd find a better way to do this.
+  for (let attempt = 0; attempt < 3; attempt++) {
+    try {
+      await page.keyboard.press('ControlOrMeta+Enter')
+      const responseEditor = page.locator('#response-editor .cm-content[role="textbox"]')
+      await expect(responseEditor).toBeVisible()
+
+      const responseText = await responseEditor.textContent()
+      const responseJson = JSON.parse(responseText?.trim() || '{}')
+
+      expect(responseJson).toEqual({})
+      break
+    } catch (error) {
+      if (attempt === 2) throw error
+    }
+  }
```

Comment:
@bradleydwyer I added this helper to fix the `CommandPalette` tests, which might help here too: https://github.com/TBD54566975/ftl/blob/main/frontend/console/e2e/helpers.ts#L27-L35.

You can see how it's used here: https://github.com/TBD54566975/ftl/blob/main/frontend/console/e2e/command-palette.spec.ts#L16-L28